app/src/main/java/fr/iut_rodez/pathpilot_android_client/signup/SignUp.java

```diff
@@ -1,16 +1,24 @@
-package fr.iut_rodez.pathpilot_android_client;
+package fr.iut_rodez.pathpilot_android_client.signup;
 
+import android.content.Intent;
 import android.os.Bundle;
 import android.util.Log;
-import android.view.View;
 import android.widget.EditText;
 import android.widget.TextView;
-import android.widget.Toast;
 
 import androidx.activity.EdgeToEdge;
 import androidx.appcompat.app.AppCompatActivity;
 
-public class MainActivity extends AppCompatActivity implements View.OnClickListener {
+import fr.iut_rodez.pathpilot_android_client.R;
+import fr.iut_rodez.pathpilot_android_client.login.Login;
+import fr.iut_rodez.pathpilot_android_client.signup.SignUpService.SignUpInput;
+import fr.iut_rodez.pathpilot_android_client.util.Popup;
+
+public class SignUp extends AppCompatActivity {
```

Comment:
Add Javadoc class comments

Reply:
Fix in https://github.com/SAE-S5-01-Developpement-avance/PathPilot-android-client/pull/44/commits/f495a2fcc43e0fc3d934ae46d71b9af6a0856994

Reply:
Fix in https://github.com/SAE-S5-01-Developpement-avance/PathPilot-android-client/pull/44/commits/f495a2fcc43e0fc3d934ae46d71b9af6a0856994